puppetdb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "6.10.1"
+appVersion: 6.10.1
 description: A Helm chart for PuppetDB
 name: puppetdb
-version: 0.11.3
+version: 1.0.0
```

Comment:
This chart is made to be used exclusively with a specific version of the puppetserver chart right?

In that case it should be really clear which version of the puppetdb chart is compatible with which version of the puppetserver chart.

Reply:
Okay, but how would you like to define that ?